db/360/SceSblPostSsMgr.yml

```diff
@@ -62,6 +62,8 @@ modules:
           ksceSblUtMgrHasNpTestFlag: 0x9FD835B0
           ksceSblUtMgrHasStoreFlag: 0x9D2E2D39
           ksceSblUtMgrResetUtokenFile: 0x1FF699DD
+          ksceSblRSA2048CreateSignature: 0x07BAD056
+          ksceSblRSA2048VerifySignature: 0x3DB69911
```

Comment:
these two nid's location have to move to up.

Reply:
I was forgot sort